docs/onboard-to-techpass.md

```diff
@@ -0,0 +1,18 @@
+# Onboard to TechPass
+
+Users can onboard to TechPass as **public officers** or **vendors**.
+
+Refer to the following table to identify your onboarding persona:
+
+| Persona| Description | <div style="width:210px">Examples</div> |
+|----| ------------- |:-------------:|
+| **Vendor** | Users who do not have a WOG account. These users may have an email address provided by the vendor organisation or it may belong to specific domains such as<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- dsta.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- dsta-wog.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- mindef.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- defence.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- gebiz.gov.sg<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- sps.gov.sg<br><br>**Note**: Email domain is the part of an email address that comes after the “@” symbol. For example, if your email address is john_doe@sps.gov.sg, then **sps.gov.sg** is your email domain.| - john_doe@ncs.com.sg<br>- john_doe@accenture.com.sg<br>- john_doe@dsta.gov.sg<br>- john_doe@gebiz.gov.sg  |
```

Comment:
sps.gov.sg
is this list from JunJie?

Reply:
Updated. I have included a statement that vendors can't use their personal email ids in the table.